app/src/components/inputs/inputUserSearch/inputUserSearch.scss

```diff
@@ -1,13 +1,61 @@
-.select2-search-users{
-  :global{
+.select2-search-users {
+  :global {
     background-color: $COLOR--white-two;
-    .Select-control, .Select-menu-outer{
+    width: inherit;
+    .error {
+      .Select-input {
+        input {
+          border-color: $COLOR--orange-red;
+        }
+      }
+      &:hover {
+        .Select-input {
+          input {
+            border-color: $COLOR--orange-red;
+          }
+        }
+      }
+    }
+    .Select {
+      width: inherit;
+      &:hover {
+        input {
+          border-color: $COLOR--gray-47;
+        }
+      }
+    }
+    .Select-control {
+      position: relative;
+    }
+    .Select-menu-outer {
+      position: absolute;
+      width: inherit;
+      z-index: 100;
+    }
+    .Select-multi-value-wrapper {
       position: relative;
     }
-    .Select-placeholder{
-      display: none;
+    .Select-placeholder {
+      line-height: 15px;
+      position: absolute;
+      top: 3px;
```

Comment:
Please write style properties in correct order. Check all following cases.
https://learn.javascript.ru/css-format